setup.py

```diff
@@ -32,7 +32,7 @@
 def get_all_extras() -> Dict:
 
     cli_deps = [
-        "click>=7.0.0,<8.0.0",
+        "click==8.0.2",
```

Comment:
Same as above

Reply:
Usually, this is strongly discouraged. Deps should be defined with wide ranges for maximum usability. But I gather that's not possible in this case?